README.md

```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/IBM/watson-calorie-counter.svg?branch=master)](https://travis-ci.org/IBM/watson-calorie-counter)
 ![Bluemix Deployments](https://deployment-tracker.mybluemix.net/stats/19a3d5f0934a2950b8c1b2b813b98d30/badge.svg)
+<!--![Bluemix Deployments](https://metrics-tracker.mybluemix.net/stats/538ed648bda50b9f22d64a8be817840f/badge.svg) -->
 
```

Comment:
'538..' is the id for news-alerting. Not sure if this is temp placeholder id or not.

Reply:
@rhagarty yeah, i have to run this once to generate the ID. merge and then i'll update?

Reply:
wll do.